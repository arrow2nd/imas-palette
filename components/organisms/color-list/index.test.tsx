import { render } from '@testing-library/react'

import ColorList from './index'

describe('ColorList', () => {
  test('見た目が変化していないか', () => {
    const { container } = render(<ColorList current="" onChange={jest.fn()} />)

    expect(container).toMatchSnapshot()
  })
})
