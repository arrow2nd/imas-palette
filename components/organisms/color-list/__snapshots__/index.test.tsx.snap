// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorList 見た目が変化していないか 1`] = `
<div>
  <div
    class="flex flex-wrap mt-8 justify-center items-center"
  >
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-gray-600"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 bg-gradient-to-r from-red-500 to-purple-500"
        data-testid="color-sample-circle"
        title="全て"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(255, 0, 0);"
        title="赤"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(255, 165, 0);"
        title="橙"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(255, 255, 0);"
        title="黄"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(144, 226, 0);"
        title="黄緑"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(0, 128, 0);"
        title="緑"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(0, 255, 255);"
        title="水色"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(0, 0, 255);"
        title="青"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(128, 0, 128);"
        title="紫"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(255, 255, 255);"
        title="白"
      />
    </div>
    <div
      class="mx-1 p-0 rounded-full cursor-pointer border-2 border-transparent"
      data-testid="colorlist-button"
    >
      <div
        class="w-8 h-8 rounded-full border-2 false"
        data-testid="color-sample-circle"
        style="background-color: rgb(0, 0, 0);"
        title="黒"
      />
    </div>
  </div>
</div>
`;
